### PR TITLE
New package: kvirc-4.2.0

### PR DESCRIPTION
**Pull request by thypon**
#998

### CODE REVIEW

**Comment by thypon** [comment]
#998

File: srcpkgs/kvirc/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'kvirc'
+pkgname=kvirc
+version=4.2.0
+revision=1
+build_style=cmake
+depends="python<3.0.0"
```
Comment (Gottox) [inline srcpkgs/kvirc/template]: python points to python2.7 at the moment. Depend on the python3 package instead.

Comment (thypon) [inline srcpkgs/kvirc/template]: Python2 is an optdepends for plugins. 

There is a misunderstanding. KVirc depends on python2 (i put <3.0.0 meaning all that is less thant 3.0.0, then the opposite of python3).

http://www.kvirc.net/?id=install here the install guide. Maybe i could move python python,perl,ada,c,c++,fortran,java,objc,lto to optdepends.

Comment (Gottox) [inline srcpkgs/kvirc/template]: Ah, missread that. Keep in mind, that python is not installed at buildtime, therefore kvirc will be build without python support:
```
-- Cryptography:
--    Crypt Support And Engines   : Yes
--    OpenSSL Support             : Not available
-- Scripting:
--    Perl Support                : Yes, used for scripting+doc
--    Python Support              : No, not found.
```

**Comment by xtraeme** [comment]
In that case you should also add `libressl-devel` to `makedepends`.

**Comment by thypon** [comment]
Doesn't perl-devel exists in repositories?

**Pull request merged**